schemas/team/summary.yml

```diff
@@ -0,0 +1,44 @@
+type: object
+
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The team's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the team description
+
+  title:
+    type: string
+    readOnly: true
+    description: |
+      The human-readable title of the team
+
+  members:
+    type: array
+    readOnly: true
+    items:
+      $ref: '#/definitions/user/summary'
+    description: |
+      Summaries of users that are part of this team.
+
+  organization:
+    $ref: '#/definitions/organization/summary'
+    readOnly: true
+    description: |
+      Summary of the organization that owns this team.
```

Comment:
At the moment I've got the docs just to show the ID of the organization. Either this API could do another request to get the details of the organization, or we could change the seed-auth-api docs.